js/index.js

```diff
@@ -16,6 +40,26 @@ class StyleManager {
 
 const styleManager = new StyleManager();
 
-DOMSelect('#refresh').onclick = (e) => {
-  styleManager.apply('p{ color: red }');
+const setupEditor = () => {
+  const node = DOMCreate('div');
+  node.id = 'editor';
+  node.style = 'margin:0;padding:0;'
+  DOMSelect('body').appendChild(node);
+
+  ace.config.set('basePath', 'https://pagecdn.io/lib/ace/1.4.8')
```

Comment:
I am learning from you how to use the AceEditor library (with CDN)

Reply:
Thanks! I had to learn in record time how to use Ace. 😁